cmd/output.go

```diff
@@ -35,7 +35,7 @@ func End(format string, args ...interface{}) {
 	os.Exit(0)
 }
 
-func Fatal(err error, args ...interface{}) {
+func Error(err error, args ...interface{}) {
```

Comment:
I needed non fatal errors and wanted to ensure they used the same formatting that had been done with fatal